inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -846,4 +874,66 @@ InferenceEngine::Blob::Ptr MKLDNNConvolutionNode::createInternalBlob(InferenceEn
     return internalBlob;
 }
 
+void MKLDNNConvolutionNode::prepareParams() {
+    mkldnn::primitive_attr attr;
+    addZeroPoints(attr);
+    setPostOps(attr, true);
```

Comment:
We have to move this code to the initialization phase and store the produced attributes as a class member.